### PR TITLE
Kcache local

### PR DESCRIPTION
**Pull request by cosunae**

### CODE REVIEW

**Comment by cosunae** [comment]
launch jenkins

**Comment by cosunae** [comment]
launch jenkins

**Comment by cosunae** [comment]
launch jenkins

File: test/integration-test/CodeGen/local_kcache_benchmark.cpp [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+//
+//  This file is distributed under the MIT License (MIT).
+//  See LICENSE.txt for details.
+//
+//===------------------------------------------------------------------------------------------===//
+#define GRIDTOOLS_CLANG_GENERATED 1
+#define GRIDTOOLS_CLANG_HALO_EXTEND 3
+#define GT_VECTOR_LIMIT_SIZE 30
+
+#undef FUSION_MAX_VECTOR_SIZE
```
Comment (twicki) [inline test/integration-test/CodeGen/local_kcache_benchmark.cpp]: we don't need all of that

File: test/integration-test/CodeGen/local_kcache_benchmark.cpp [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+TEST(local_kcache, test) {
+  domain dom(Options::getInstance().m_size[0], Options::getInstance().m_size[1],
+             Options::getInstance().m_size[2]);
+  dom.set_halos(halo::value, halo::value, halo::value, halo::value, 0, 0);
+
+  verifier verif(dom);
+
+  meta_data_t meta_data(dom.isize(), dom.jsize(), dom.ksize() + 1);
+  storage_t a_gt(meta_data, "a_gt"), a_naive(meta_data, "a_naive"), b_naive(meta_data, "b"), b_gt(meta_data, "b_gt");
+  storage_t c_gt(meta_data, "c_gt"), c_naive(meta_data, "c_naive"), d_naive(meta_data, "d"), d_gt(meta_data, "d_gt");
```
Comment (twicki) [inline test/integration-test/CodeGen/local_kcache_benchmark.cpp]: call the a_opt etc if we're already moving in that direction

**Pull request merged**